packages/experimental/Shimmer/package.json

```diff
@@ -16,16 +13,17 @@
     "build": "fluentui-scripts build",
     "just": "fluentui-scripts",
     "clean": "fluentui-scripts clean",
-    "lint": "fluentui-scripts eslint",
-    "test": "fluentui-scripts jest",
     "update-snapshots": "fluentui-scripts jest -u",
     "verify-api": "fluentui-scripts verify-api-extractor",
     "update-api": "fluentui-scripts update-api-extractor"
   },
   "dependencies": {
     "@fluentui-react-native/framework": "0.5.1",
-    "@fluentui-react-native/adapters": ">=0.6.0 <1.0.0",
-    "@fluentui-react-native/tokens": ">=0.8.0 <1.0.0"
+    "@uifabricshared/foundation-compose": "^1.8.1",
```

Comment:
Do we need this dependency still? It wouldn't just be "@fluentui-react-native/framework"?

Reply:
Yep, no longer needed.